articles/110_interface_definition.md

```diff
@@ -70,11 +70,15 @@ The following primitive types are defined:
 * `string`
 
 <div class="alert alert-warning" markdown="1">
+  **Question:** in ROS 1, `string` was ASCII. The Python 3 `str` type is UTF-8. What is the DDS `string` type?
```

Comment:
In ROS 1 as well as in ROS 2 the string message type is only storing a sequence of bytes. Any encoding is user land responsibility.